conversions/vrep_conversion.cpp

```diff
@@ -0,0 +1,85 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <cstring>
+#include "mrpt/obs/CObservation2DRangeScan.h"
+#include "vrep_conversion.h"
+#include "mrpt/math/CQuaternion.h"
+#include "mrpt/obs/CObservationOdometry.h"
+
+using namespace mrpt::obs;
+using namespace mrpt::poses;
+using namespace mrpt::math;
+
+namespace vrep_bridge
+{   /**
+    *Converts VREP laser scans into a CObservation2DRangeScan class object. It has the following arguments:
```

Comment:
Just noticed this.

The doxygen comments should be put in the declaration of the methods i.e., in the `.h` file